File: listenmoe-api/src/main/java/me/echeung/listenmoeapi/endpoints/Jpop.java
package me.echeung.listenmoeapi.endpoints;

import lombok.AccessLevel;
import lombok.NoArgsConstructor;

@NoArgsConstructor(access = AccessLevel.PRIVATE)
public class Jpop extends Library {

    public static final Library INSTANCE = new Jpop();

    public static final String SOCKET = "wss://listen.moe/gateway";
    public static final String STREAM_MP3 = "https://listen.moe/fallback";

}

File: app/src/main/java/me/echeung/moemoekyun/client/api/v4/library/Jpop.kt
package me.echeung.moemoekyun.client.api.v4.library

class Jpop private constructor() : Library(
        NAME,
        "wss://listen.moe/gateway",
        "https://listen.moe/fallback") {
    companion object {
        const val NAME = "jpop"

        val INSTANCE: Library = Jpop()
    }
}
